dist/rpm/openQA.spec

```diff
@@ -27,7 +27,7 @@
 %{nil}
 %endif
 # Run tests on openSUSE Tumbleweed and supported openSUSE Leap versions
-%if 0%{?suse_version} >= 1550 || ( 0%{?is_opensuse} && 0%{?sle_version} >= 150100 )
+%if 0%{?suse_version} >= 1550 || ( 0%{?is_opensuse} && 0%{?suse_version} >= 1500 )
```

Comment:
suse_version 1550 does not exists, see https://en.opensuse.org/openSUSE:Packaging_for_Leap#RPM_Distro_Version_Macros

Reply:
I did see that but I thought that it allows to take the _minor_. I did not change that part

Reply:
it should be `%if 0%{?suse_version} >= 1550 || ( 0%{?is_opensuse} && 0%{?sle_version} >= 150600 )`  according to command above ... `suse_version`  contains only minimal product so which is +- disconnected with rest of versioning system

`sle_version` on other side is contains both see/leap main version and service pack